augur/curate/abbreviate_authors.py

```diff
@@ -0,0 +1,88 @@
+"""
+Abbreviates a full list of authors to be '<first author> et al.' of the NDJSON
+record from stdin and outputs modified records to stdout.
```

Comment:
_nit_

stdin/stdout is not _always_ true because `augur curate` has many I/O options. I'd drop any references to I/O in subcommand docs since that is centralized in shared parser.